ui/tx/details/TxRevertReason.tsx

```diff
@@ -12,7 +12,14 @@ const TxRevertReason = (props: Props) => {
   const bgColor = useColorModeValue('blackAlpha.50', 'whiteAlpha.50');
 
   if ('raw' in props) {
+    const hexRegexp = /^(?:0x)?[\da-fA-F]+$/;
```

Comment:
maybe it is better to store this RegExp here `lib/regexp.ts` in order to reuse it later in other parts of the code base